home-data-migration/src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <include file="v-0.0.1/db.changelog-v.0.0.1.xml" relativeToChangelogFile="true"/>
```

Comment:
Let's omit prefix `v-` from version naming. I believe that it is obvious what is it about.

Reply:
Done